sessions/react-custom-hooks/advanced-fetch_solution/README.md

```
@@ -0,0 +1,50 @@
+# Advanced Fetch
+
+In this exercise you will enhance the `useFetch` hook from the session with a loading state as well as a simple `cache`.
+
+## Tasks
+
+Take a look at the project files. You find the code from the previous session. Let's start by adding a loading state to our `useFetch` hook.
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)